README.md

```diff
@@ -5,7 +5,7 @@ Docker image to load user profiles into CCD.
 ## Building
 
 Any commit or merge into master will automatically trigger an Azure ACR task. This task has been manually
-created using `./bin/deploy-acr-task.sh`. The task is defined in `acr-build-task.yaml`. Note: you will need a GitHub personal token defined in `GITHUB_TOKEN` environment variable to run deploy script (https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line).
+created using `./bin/deploy-acr-task.sh`. The task is defined in `acr-build-task.yaml`. Note: you will need a GitHub personal token defined in `GITHUB_TOKEN` environment variable to run deploy script (https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line). The token is for setting up a webhook so Azure will be notified when a merge or commit happens.
```

Comment:
what scope is needed?

Reply:
GitHub scope? I don't know 🙈 I'm gonna guess read access. Not sure how i can test this. I'll check (google)...

Reply:
I assume repo admin, but would be good if you could test it out